digdag-standards/src/main/java/io/digdag/standards/operator/pg/PgConnection.java

```diff
@@ -103,7 +103,7 @@ public void cleanup()
             executeStatement("delete old query status rows from " + escapeIdent(statusTableName) + " table",
                     String.format(ENGLISH,
                         "DELETE FROM %s WHERE query_id = ANY(" +
-                        "SELECT query_id FROM %s WHERE completed_at < now() - interval '%d' seconds" +
+                        "SELECT query_id FROM %s WHERE completed_at < now() - interval '%d seconds'" +
```

Comment:
`interval '1' seconds` is the SQL standard. `interval '1 second'` is a PostgreSQL extension. I'm also surprised when I saw that query issued by Tableau through Prestogres but it's for sure.
See http://savage.net.au/SQL/sql-92.bnf.html#interval literal